src/order/complete_lattice.lean

```diff
@@ -818,6 +818,17 @@ supr_subtype
 lemma infi_subtype'' {ι} (s : set ι) (f : ι → α) : (⨅ i : s, f i) = ⨅ (t : ι) (H : t ∈ s), f t :=
 infi_subtype
 
+lemma bsupr_const {ι : Sort*} {a : α} {s : set ι} (hs : s.nonempty) :
+  (⨆ (i ∈ s), a) = a :=
```

Comment:
```suggestion
  (⨆ i ∈ s, a) = a :=
```

Reply:
OK, I'm already at the back of the action queue.